mechanics/RELEASE-SCHEDULE.md

```diff
@@ -22,8 +22,6 @@ With that said, it can be useful to have a list of when future releases will hap
 
 | Release | Date       | EOL        | Min K8s Version | Notes |
 | ------- | ---------- | ---------- | --------------- | ----- |
-| 1.10    | 2023-04-25 | 2023-10-31 | 1.24            |       |
-| 1.11    | 2023-07-25 | 2024-01-30 | 1.25            |       |
 | 1.12    | 2023-10-24 | 2024-04-30 | 1.26            |       |
 | 1.13    | 2024-01-23 | 2024-07-30 | -               |       |
```

Comment:
Do we know what min version we want to support for 1.13+?

Reply:
Likely `1.27` it seems to me.